src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -76,6 +76,11 @@
             <tag name="serializer.normalizer" priority="1000" />
         </service>
 
+        <service id="serializer.normalizer.uid" class="Symfony\Component\Serializer\Normalizer\UidNormalizer">
+            <!-- Run before serializer.normalizer.object -->
+            <tag name="serializer.normalizer" priority="-880" />
+        </service>
```

Comment:
Should be moved to the new PHP config file.
For the priority, I would use -915 to be consistent with the other very specialized normalizers.